lib/member_page.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+require 'scraped_page'
+
+class MemberPage < ScrapedPage
+  def initialize(url:, strategy: Strategy::LiveRequest.new, entry:)
```

Comment:
Having to override the `initialize` method is a warning sign that this probably isn't the best way to go about this. 

It's also not clear what `entry` is in this context (i.e. that might be a plausible name for this in the caller, but it doesn't seem to convey much here)